test/kubernetes_test.go

```diff
@@ -250,8 +250,11 @@ var dnsTestCases = []test.Case{
 	},
 	{
 		Qname: "*.svc-1-a.test-1.svc.cluster.local.", Qtype: dns.TypeSRV,
-		Rcode:  dns.RcodeSuccess,
-		Answer: []dns.RR{},
+		Rcode: dns.RcodeSuccess,
+		Answer: []dns.RR{
+			test.SRV("svc-1-a.test-1.svc.cluster.local.	303	IN	SRV	0 50 443 svc-1-a.test-1.svc.cluster.local."),
```

Comment:
The answers should (I think), have the question in them ... e.g. 
`test.SRV("svc-1-a.test-1.svc.cluster.local.	303	IN	SRV	0 50 443 svc-1-a.test-1.svc.cluster.local.")`
should instead be ...
`test.SRV("*.svc-1-a.test-1.svc.cluster.local.	303	IN	SRV	0 50 443 svc-1-a.test-1.svc.cluster.local.")`